crates/viewer/re_viewer/src/app.rs

```diff
@@ -328,6 +328,23 @@ impl App {
             command_sender.send_ui(UICommand::ExpandBlueprintPanel);
         }
 
+        creation_context.egui_ctx.on_end_pass(
+            "remove copied text formatting",
+            Arc::new(|ctx| {
+                ctx.output_mut(|o| {
+                    #[expect(deprecated)]
+                    if !o.copied_text.is_empty() {
+                        o.copied_text = re_format::remove_number_formatting(&o.copied_text);
+                    }
+                    o.commands.iter_mut().for_each(|event| {
+                        if let egui::output::OutputCommand::CopyText(text) = event {
+                            *text = re_format::remove_number_formatting(text);
+                        }
+                    });
```

Comment:
style: what's wrong with a good old fashion for loop? 🤷 

```suggestion
                    for command in &mut o.commands {
                        if let egui::output::OutputCommand::CopyText(text) = command {
                            *text = re_format::remove_number_formatting(text);
                        }
                    }
```